awx/main/conf.py

```diff
@@ -831,6 +831,56 @@
     category_slug='system',
 )
 
+register(
+    'AWX_CALLBACK_PROFILE',
+    field_class=fields.BooleanField,
+    label=_('Debug callback timing'),
+    default=False,
+    help_text=_('Debug callback code timing'),
```

Comment:
This has a lot more explaining to do than "debug callback code timing", because this enables listening for the `SIGUSR1` signal, and if it receives it, it will then start profiling. But this means they need to shell in, find the process of interest, and send the signal.

But even _if_ they are competent enough to do that, I don't think this will work right, because the signal is registered at the start of the `work_loop` only if the setting is turned on. So then you still have to restart services for this to work.